crates/modelardb_server/src/storage/compressed_data_manager.rs

```diff
@@ -517,6 +518,7 @@ mod tests {
     use datafusion::arrow::datatypes::{ArrowPrimitiveType, Field, Schema};
     use futures::StreamExt;
     use modelardb_common::metadata::model_table_metadata::ModelTableMetadata;
+    use modelardb_common::metadata::try_new_sqlite_table_metadata_manager;
```

Comment:
Maybe only import `modelardb_common::metadata` and use the function as  `metadata::try_new_sqlite_table_metadata_manager()` for consistency.

Reply:
Fixed.